content/community/contributing/handbook/test/test-strategy/_index.md

```diff
@@ -53,7 +53,7 @@ To be be able to easily and effectively test on all levels, it can be necessary
 During sprint planning, the product owner, the developers, and the test developer agree on an approach for teting on individual user stories that are pulled into the sprint. A plan is then created for how the responsible role will test the required functionality, and sub tasks are created on the different user stories. It is possible, and likely, that a user story will contain tasks to develop tests on all the testing levels specified above. When a test is written and completed, it can be integrated into a testing build. 
 
 **Naming convention for element ids'**
-Testcafe's test api has different methods of instantiating Selector objects that can be then used in tests. The easiest and most robust method of defining a selector in testcafe is with a css id selector. The idea behind using css id's is to avoid finding elements by their relative location in the DOM, or by the element's inner texts, as tests become brittle with these selectors due to design changs. The id's must be unique, to ensure the uniqueness of a Selector, the following naming convention for element id's is proposed: Id's must be tekstbased and written in camelCase, id's follow a logical location the Altinn Studio system, such as; "Pagelocation Submenu Elementtype Descriptiveword". The text box element id under the GUI creator could then be assigned the following id: "designerSkjemakomponenterKnappTekstområde".
+Cypress has different methods of getting elements from the DOM. The easiest and most robust method of defining a selector is with a css id selector. The idea behind using css id's is to avoid finding elements by their relative location in the DOM, or by the element's inner texts, as tests become brittle with these selectors due to design changs. The id's must be unique, to ensure the uniqueness of a Selector, the following naming convention for element id's is proposed: Id's must be tekstbased and written in camelCase, id's follow a logical location the Altinn Studio system, such as; "Pagelocation Submenu Elementtype Descriptiveword". The text box element id under the GUI creator could then be assigned the following id: "designerSkjemakomponenterKnappTekstområde".
```

Comment:
```suggestion
Cypress has different methods of getting elements from the DOM. The easiest and most robust method of defining a selector is with a css id selector. The idea behind using html-id's is to avoid finding elements by their relative location in the DOM, or by the element's inner texts, as tests become brittle with these selectors due to design changs. The id's must be unique, to ensure the uniqueness of a Selector, the following naming convention for element id's is proposed: Id's must be tekstbased and written in camelCase, id's follow a logical location the Altinn Studio system, such as; "Pagelocation Submenu Elementtype Descriptiveword". The text box element id under the GUI creator could then be assigned the following id: "designerSkjemakomponenterKnappTekstområde".
```